File: codeWars/katas/square_every_digit/square_every_digit.py
# Square Every Digit


def square_digits(num):
    numString = str(num)
    result = ""

    for i in numString:
        result += str(int(i) ** 2)

    return result


print(square_digits(9119))  # Output: 811181
print(square_digits(765))  # Output: 493625

File: codeWars/katas/square_every_digit/square_every_digit.js
// Square Every Digit


function squareDigits(num) {
    const numString = num.toString();
    let result = "";

    for (let i = 0; i < numString.length; i++) {
        result += numString[i] ** 2;
    }

    return parseInt(result);
}


console.log(squareDigits(9119)); // Output: 811181
console.log(squareDigits(765)); // Output: 493625
